app/javascript/app/document-capture/components/documents-step.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FileInput from './file-input';
+import useI18n from '../hooks/use-i18n';
+
+/**
+ * List of supported mime types for document file upload.
+ *
+ * @type {string[]}
+ */
+const SUPPORTED_IMAGE_MIME_TYPES = ['image/jpeg', 'image/bmp', 'image/png', 'image/tiff'];
```

Comment:
Are we supporting HEIC? Or is this only intended for desktop?

Reply:
> Are we supporting HEIC? Or is this only intended for desktop?

I'd have to double-check the requirements for the image. This was primarily based on a [design](https://gsa-tts.slack.com/archives/CEUQ9FXNJ/p1595534805129800?thread_ts=1595427041.108700&cid=CEUQ9FXNJ) which included a hint text "Must be a JPG, BMP, PNG, or TIFF". The hint would only be shown for desktop, but I assume the input would be reused for both desktop and mobile, so the `accept` assigned to the input should either include all possible options, or at least be device-aware enough to allow additional types if they can be supported.

Reply:
The documentation isn't entirely clear here. After chatting with @anniehirshman-gsa , the hint from the design is pulled from existing screens. There's a desire to support HEIC, though unclear as of yet if Acuant actually supports it (see LG-3026).

After digging in the SDK source, I did find this, though it's still not clear to me if `Module.noImageDecoding` makes a difference here. I'm not sure what impact that makes:

https://github.com/Acuant/JavascriptWebSDKV11/blob/54d6b4ec7ee38a0e98abc622caf2d98e9b454e2e/SimpleHTMLApp/webSdk/dist/AcuantJavascriptWebSdk.js#L4457

There's an argument to be made that we could just avoid `accept` altogether, let the user upload whatever they have, and deal with any errors afterward. The advantage here is to try to preempt those error scenarios. In the case of mobile upload though, since the user is not as likely to be using the file picker, we could opt to make this a device-specific assignment: Restrict file types on desktop, allow all types on mobile.

Since we don't have device detection implemented yet, it'd probably be something to follow-up on.

That being said, I did notice in the course of investigating this that there are apparently two possible MIME types for JPEG images (`image/jpg` and `image/jpeg`), which should be reflected here.

Reply:
> The documentation isn't entirely clear here. After chatting with @anniehirshman-gsa , the hint from the design is pulled from existing screens. There's a desire to support HEIC, though unclear as of yet if Acuant actually supports it (see LG-3026).

As a compromise to all of the above, in e6b0ed5 I updated the `accept` attribute to `image/*`, to benefit from filtering file picker results to images, while not being so prescriptive as to list every supported file MIME type.